lib/node_modules/@stdlib/array/fixed-endian-factory/lib/main.js

```diff
@@ -723,6 +723,54 @@ function factory( dtype ) { // eslint-disable-line max-lines-per-function, stdli
 		return out.join( ',' );
 	});
 
+	/**
+		* return sliced TypedArray.
+		*
+		* @private
+		* @name slice
+		* @memberof TypedArray.prototype
+		* @type {Function}
+		* @throws {TypeError} `this` must be a typed array instance
+		* @returns {TypedArray} string representation
+		*/
+
+	setReadOnly( TypedArray.prototype, 'slice', function slice() {
+		var value1;
+		var value2;
+		var nargs;
+		var temp;
+		var out;
+		var buf;
+		var i;
+		value1 = arguments[0];
+		value2 = arguments[1];
+		nargs = arguments.length;
+		if (value2 === null || nargs === 1) {
+			value2 = this._length;
+		}
+		if ( nargs < 0 || nargs > 2 || !isNonNegativeInteger( value1 ) || !isNonNegativeInteger( value2 ) ) {
+			throw new TypeError( 'invalid arguments.' );
+		}
+		if ( value1 >= this._length || value2 > this._length || value1 >= value2 ) {
+			throw new RangeError( 'invalid range of arguments.' );
+		}
+		if ( !isTypedArray( this ) ) {
+			throw new TypeError(format('invalid invocation. `this` is not %s %s.', CHAR2ARTICLE[ dtype[0]], CTOR_NAME));
+		}
+		temp = [];
```

Comment:
You don't need this temporary array. As in https://github.com/stdlib-js/stdlib/blob/4b1d77d2bd001d5970ce93230765a579fb41349d/lib/node_modules/%40stdlib/array/bool/lib/main.js#L1880, you can create a new instance and then copy elements directly to the target array DataView buffer.